docs/configure/user-interface.md

```diff
@@ -0,0 +1,392 @@
+
+For example, the popular Actions addon uses [react-inspector](https://github.com/xyc/react-inspector/blob/master/src/styles/themes/chromeLight.js) which has themes of its own. Supply additional theme variables to style it like so:
+
+```js
+
+addonActionsTheme: {
+  ...chromeLight,
+  BASE_FONT_FAMILY: typography.fonts.mono,
+  BASE_BACKGROUND_COLOR: 'transparent',
+}
+```
+
+### Using the theme for addon authors
+
+Reuse the theme variables above for a native Storybook developer experience. The theming engine relies on [emotion](https://emotion.sh/), a CSS-in-JS library.
+
+```js
+import { styled } from '@storybook/theming';
+```
+
+Use the theme variables in object notation:
+
+```js
+const Component = styled.div(({ theme }) => ({
+  background: theme.background.app,
+  width: 0,
+}));
+```
+
+Or with template literals:
+
+```js
+const Component = styled.div`
+  background: `${props => props.theme.background.app}`
+  width: 0;
+`;
+```
+
+### Storybook addons
+
+A key strength of Storybook is its extensibility. Storybook aims to be your tool of choice for developing components in the browser no matter what choice of UI framework your team has made. Another way that Storybook can fit your team’s development workflow is through addons that allow you to build better and more robust components.
+
+Addons have been a part of Storybook since the very beginning and are so integral to the way Storybook works that many of the features described in this documentation are implemented as addons! These addons are installed out of the box with [essentials](../essentials). 
+
+#### Addon features
+
+The most obvious thing addons affect in Storybook is the UI of Storybook itself. Within the UI the [toolbar]() and [addons panel]() are the two chief places addons will appear. 
+
+<div style="background-color:#F8FAFC">
+TODO: add image mentioned in the SB 6.0 needs to be further vetted
+</div>
+
+Addons can also hook into the rendering of your story in the preview pane via injecting their own [decorators](../writing-stories/decorators).
+
+Finally, addons can affect the build setup of Storybook by injecting their own webpack configuration to allow the use of other tools in Storybook. Addons that do only this are often referred to as [presets](../presets/introduction).
+
+### Essential, core and community addons
+
+There are many, many Storybook addons, but they can be roughly categorized into three areas:
+
+- Essential addons are core-team developed addons that are considered core to the out-of-the-box user experience of Storybook and thus ship by default with new Storybook installations.
+- Core addons are the addons that are developed by the core team with efforts made to keep them in sync with the development of Storybook itself, and to make sure they are written in idiomatic ways as templates for other addons. They can be found within the Storybook monorepo,
```

Comment:
Link to mono repo?

Reply:
It's linked to the monorepo, more specifically to the addons folder to be more exact.